src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/StringModule.fs

```diff
@@ -41,8 +41,8 @@ type StringModule() =
         let e6 = String.concat "foo" ["bar"]
         Assert.AreEqual("bar", e6)
         
-        let e7 = String.concat "foo" ["bav";"baz"]
-        Assert.AreEqual("bavfoobaz", e7)
+        let e7 = String.concat "foo" ["bar";"baz"]
+        Assert.AreEqual("barfoobaz", e7)
```

Comment:
but it's not "barf". it's "barfoobaz" ;-)
I don't thing this word exists at all. If your tool is complaining about infixes with barf then the german word "barfrau" (female barkeeper) would never be used inside of MS!?